### ISSUE TITLE
Preview for new posts does not show imported image

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-Android#8755]
This happens on both debug and production builds. Production build was 11.3.

### Expected behavior

Previewing a new post should show the image imported from the gallery.

### Actual behavior

An HTML `<img>` referencing the local path of the image is shown instead.

![16c2986cda8dc02b4eac1a796968e65a](https://user-images.githubusercontent.com/198826/49699099-318a3580-fb8a-11e8-8e8c-c05295492f11.gif)

However, if you leave the editor and go edit the post again, the preview works fine.

![b3abc151be30ad6e38a8994dbfe8b977](https://user-images.githubusercontent.com/198826/49699103-3a7b0700-fb8a-11e8-9dad-defa4a0e3cae.gif)

#### Setup

This issue shows up even in production version 11.3. For debug builds, I tested this on `develop` at commit 01de4c62ec using Android Emulator Pixel API 27 Android 8.1.

#### Steps

1. Create a new post using the _New Post_ button on the bottom tab bar.
2. Add a title and a text on the body.
3. While the soft keyboard is shown, click on the + button.
4. Pick a photo from the shown gallery.
5. After the photo is shown in the post's content, click on the options button at the top right.
6. On the shown options menu, click on _Preview_.

The image will not be shown. An HTML with an `<img>` tag is shown instead.

**Comment by rachelmcr** [comment]
Good catch! This looks like a regression with the local preview (which should show the image, not the `img` tag) and another argument to work on preview improvements discussed in https://github.com/wordpress-mobile/WordPress-Android/issues/4243.

**Comment by planarvoid** [comment]
For some reason there is a specific logic for `local drafts` which are showed differently than any other post. It is displaying the HTML code in a TextView instead of WebView. Removing this seems to fix the issue and not cause any additional problem. I'll raise a PR with the change.
